dashboard/lib/international_opt_in_people.rb

```diff
@@ -10,7 +10,7 @@ module InternationalOptInPeople
     barbados: ["The Trust for the Americas"],
     belize: ["The Trust for the Americas"],
     colombia: ["Computadores para Educar"],
-    chile: ["Fundacion Kodea", "Mineduc"],
```

Comment:
these are specific requests from the internationalization team, right?

Reply:
Correct. International have a spreadsheet where they track this updates.